### PR TITLE
Support Optional<T> column on Row class

### PR DESCRIPTION
**Pull request by moznion**
To wrap result value of nullable column (means column is not specified `NOT NULL` constraint).

### Motivation
I want to avoid NPE when accessing to nullable column.

### CODE REVIEW

**Comment by tokuhirom** [comment]
hmm..

File: tinyorm/src/main/java/me/geso/tinyorm/TableMeta.java [hunk 1]
```diff
@@ -152,6 +155,33 @@ private TableMeta(String name, List<PropertyDescriptor> primaryKeyMetas,
...
+			if (inflaters.get(propertyDescriptor.getName()) == null) {
+				inflaters.put(propertyDescriptor.getName(), new ArrayList<>());
+			}
+			if (deflaters.get(propertyDescriptor.getName()) == null) {
+				deflaters.put(propertyDescriptor.getName(), new ArrayList<>());
+			}
+
+			boolean isOptionalColumn = false;
+			boolean isOptionalLocalDate = false;
+			boolean isOptionalLocalTime = false;
```
Comment (tokuhirom) [inline tinyorm/src/main/java/me/geso/tinyorm/TableMeta.java]: hmm. this code looks horrible.

Comment (moznion) [inline tinyorm/src/main/java/me/geso/tinyorm/TableMeta.java]: I agree...
Do you have any nice ideas?

**Comment by moznion** [comment]
@tokuhirom 
Fixed to be able to mergeable.

**Comment by tokuhirom** [comment]
Great

**Pull request merged**